### ISSUE TITLE
run script inputs

### ISSUE THREAD

**Issue by jrhea** [Consensys-Incorporated/teku#662]
### Description
I need to be able to specify custom config files & output files for individual nodes.  This will allow me to script batch runs.  Here is an example of what I'd like to see:

```bash
$ sh run.sh --numNodes=4 --inputFile=/path/to/alternateConfig.$NODE_INDEX.toml --outputFile=/path/to/output.$NODE_INDEX.csv
```

**Comment by jalextowle** [comment]
Working on this

**Comment by jrhea** [comment]
resolved by #667
